modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +41,45 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. {rh-openstack-first} only: Complete the following steps to make the failover VIP addresses reachable on {rh-openstack} ports:
```

Comment:
I haven't seen something like this, which isn't to say it is wrong. I wonder if something like:
`. If you are not using {rh-openstack-first}, go to step 4. For {rh-openstack} customers, complete the following steps:`

Reply:
All the steps apply to RHOSP, but step three is especially unique.

Reply:
I understand the structure complexity, but unlike the installer docs that use ifdef/ifndef statements to carve up shared modules for multiple platforms, we do not have this structure on Networking docs that do not break networking docs down at the platform level; doing so I would imagine, would create a bigger docs set.